servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcExecutionStrategies.java

```diff
@@ -45,7 +45,9 @@ public static GrpcExecutionStrategy defaultStrategy() {
      *
      * @param executor {@link Executor} to use.
      * @return Default {@link GrpcExecutionStrategy}.
+     * @deprecated Set the executor to use on the {@link io.servicetalk.transport.api.ExecutionContext}.
```

Comment:
Let's clarify new methods here (general comment for all `@deprecated` annotations):
```
* @deprecated Set the executor to use on the {@link io.servicetalk.transport.api.ExecutionContext} using {@code executor(Executor)} methods on client/server builders.
```